src/EncompassRest/Loans/LoanObjectBoundApis.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace EncompassRest.Loans
+{
+    public sealed class LoanObjectBoundApis : LoanApis
+    {
+        public Loan Loan { get; }
+
+        public bool ReflectToLoanObject { get; set; } // What should be the default value?
```

Comment:
what is this for?

Reply:
This is for when the borrower pairs and other API's get's added. It will determine if things done in those API's should be reflected back to the `Loan` object, e.g. call Delete then delete from loan object with that id.

Reply:
Yeah, I agree the default should be false. What do you mean should there be a default at all?

Reply:
never mind. false is the safer bet.